packages/twenty-server/src/engine/metadata-modules/agent/agent-chat.controller.ts

```diff
@@ -91,4 +105,58 @@ export class AgentChatController {
       res.end();
     }
   }
+
+  @Post('files')
+  @UseInterceptors(FileInterceptor('file'))
+  async uploadFile(
```

Comment:
How does this play with existing endpoints? Attachment is too different for sure but the first step before creating an attachment was uploading the file which seems similar